project_ideas/app/admin.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from .models import Idea, Comment, User, Vote
+from .forms import CustomUserCreationForm, CustomUserChangeForm
+
+
+class CustomUserAdmin(UserAdmin):
+    add_form = CustomUserCreationForm
+    form = CustomUserChangeForm
+    model = User
+    list_display = ('email', 'username', 'is_staff', 'is_active',)
+    list_filter = ('email', 'username', 'is_staff', 'is_active',)
+    fieldsets = (
+        (None, {'fields': ('email', 'username', 'password')}),
+        ('Permissions', {'fields': ('is_staff', 'is_active')}),
+    )
+    add_fieldsets = (
+        (None, {
+            'classes': ('wide',),
+            'fields': ('email', 'username', 'password1', 'password2', 'is_staff', 'is_active')}
```

Comment:
password2 is for conforming password

Reply:
Yes it is for confirming password used while creating superuser through CLI